Classes/Mathematik.cs

```diff
@@ -22,16 +22,78 @@ public static int Fakultaet (int zahl)
             return ergebnis;
         }
 
-        public static double Potenz (double zahl, int potenz)
+        public static double Potenz(double zahl, int exponent)
         {
             double ergebnis = 1;
 
-            for (int iterator = 0; iterator < potenz; iterator++)
+            for (int iterator = 0; iterator < exponent; iterator++)
             {
                 ergebnis *= zahl;
             }
 
             return ergebnis;
         }
+
+        public static double Bruch(int zaehler, int nenner)
+        {
+            if(nenner != 0)
+            {
+                double ergebnis = Convert.ToDouble(zaehler) / Convert.ToDouble(nenner);
+                return ergebnis;
+            }
+            else
+            {
+                throw new DivideByZeroException("Division of {" + zaehler + "} by zero.");
+            }
+        }
+
+        public static double Wurzel(double zahl)
+        {
+            double x = 1;
+
+            do
+            {
+                x = (x + zahl / x) / 2;
+            } while ((x * x).ToString("F6") != zahl.ToString("F6"));
```

Comment:
würde die funktion in der while klammer eine zeile darunter setzen um es klarer darzustellen